### ISSUE TITLE
Alt tags: default to "" if none found

### ISSUE THREAD

**Issue by stscoundrel** [pixelshelsinki/pixels-images#8]
Currently can output alt tag without any content. Should be empty string in that case. Avoids accessibility/seo issue
